server/routes/index.js

```diff
@@ -37,42 +35,10 @@ connect(amqpAddress, config.rabbitmq.opts, function (err, conn) {
 
 //Check if the information requested is for the user
 
-//Check user privilege for the study
-function ensurePermission(req, res, next) {
```

Comment:
I don't understand why we're deleting this - what's the connection to the singularity removal?

Reply:
I have same question. This appears to be auth logic for the app.

Reply:
Thanks for asking.
The Singularity code writes a file called basePath and that basePath is scattered throughout the codebase and it is currently not needed. 
The ensurePermissions function was refactored and replaced to ensureStudyPermission. The ensurePermissions function that was deleted is the last bit that uses basePath in the file.
Perhaps removing this code could be part of a bigger ticket, because basePath is scattered throughout the code, and will have to be cleaned up.

Reply:
Yeah, we'd have to remove it piece by piece. There are a few pages left to be refactored that use routes/api calls using this still, but ideally by the end of this we'd have gotten rid of it.

Reply:
So far I remember, I had to insert this line in many places to comply with MGB digital security team. It was also mandated by Veracode scan. I suggest you keep it there.